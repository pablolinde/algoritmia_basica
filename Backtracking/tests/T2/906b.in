18

3
349
10
399

22
138
425
49
69
23
89
280
278
232
137
88
291
175
359
313
383
165
138
264
51
418
241

13
271
431
114
369
381
175
369
171
364
343
219
433
365

9
262
192
192
398
332
32
174
292
344

12
58
83
422
108
102
212
347
372
244
62
342
175

8
260
397
150
152
165
184
118
73

27
310
264
46
243
295
219
136
240
376
193
375
347
353
26
161
249
450
6
362
393
180
199
203
126
400
406
342

21
74
16
179
435
331
276
279
227
44
414
69
22
208
443
420
110
70
182
410
69
188

27
12
419
122
266
147
71
221
38
256
346
106
36
330
436
312
210
213
407
173
281
428
433
325
450
144
394
181

21
240
314
332
315
304
133
205
440
4
368
216
130
28
44
343
17
360
20
216
438
51

24
398
163
197
232
275
296
10
221
152
249
84
34
166
440
166
422
31
221
391
246
402
418
342
346

27
251
365
252
291
17
43
290
180
292
72
56
189
81
276
341
381
411
426
96
2
193
120
33
413
60
278
364

6
169
311
64
22
226
367

10
242
410
203
23
303
274
130
41
406
7

25
389
418
356
34
21
150
205
53
112
265
383
77
344
101
388
407
174
215
375
87
6
334
290
81
186

12
49
213
311
310
41
33
128
302
143
415
70
338

17
437
288
235
68
104
77
226
211
71
372
378
291
335
20
148
303
47

5
403
311
205
262
404
