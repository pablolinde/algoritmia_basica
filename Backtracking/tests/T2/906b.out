359 399
2303 2303
2002 2003
1108 1110
1263 1264
746 753
3450 3450
2233 2233
3378 3379
2259 2259
2951 2951
2886 2887
559 600
1019 1020
2725 2726
1176 1177
1805 1806
778 807
